### DISCUSSION TITLE
Reinstall Calculator

### DISCUSSION THREAD

**Discussion by herbert1984** [farag2/Sophia-Script-for-Windows#183] [Q&A] [answered]
I have used the script and clicked uninstall Microsoft Calculator in the checkbox. Is there a way to reinstall it? I have tried many ways but all have failed...

**Comment by Sourav2005** [comment, marked as answer]
Look up Calculator on Windows Store.

**Reply by farag2** [reply]
Of course! Run the restore function. And if you uninstalled in only from your current account it can be retored officially.
Invoke the `RestoreUWPApps` function

https://github.com/farag2/Windows-10-Sophia-Script/blob/d4e4987061eafd5fb1653c73f8a9bb4e97cd78c1/Sophia/PowerShell%205.1/Sophia.ps1#L984

If you don't know how to call the specific function, please read the instruction. It's very easy. Feel free to ask a question.

**Reply by herbert1984** [reply]
MS Store got uninstalled in the process.

**Reply by herbert1984** [reply]
Thanks for help. I read that specific apps can only be restored if removed for one user. I uninstalled for all users. Guess I have to live without calculator...

**Reply by farag2** [reply]
No. Actually, you can re-install it anyway. Install the `Microsoft.WindowsStore_8wekyb3d8bbwe.appxbundle`, `Microsoft.Services.Store.Engagement.x64.appx`, and `Microsoft.StorePurchaseApp_8wekyb3d8bbwe.appxbundle` one by one via

```powershell
Add-AppxPackage -Path path\to\file -Verbose
```
After that install all system appxbundle from the link. And then install Calculator from the Store.

https://disk.yandex.ru/d/10Ttj2IVOKQ0Og

**Reply by herbert1984** [reply]
What would the exact command line for Microsoft.WindowsStore_8wekyb3d8bbwe.appxbundle be? I have tried but am getting error messages. Sorry, I am not that good with Powershell.

**Reply by farag2** [reply]
```powershell
Add-AppxPackage -Path "The\Full\path\to\Microsoft.WindowsStore_8wekyb3d8bbwe.appxbundle" -Verbose
```

Like "D:\Downloads\Microsoft.WindowsStore_8wekyb3d8bbwe.appxbundle" And the same way for each downloaded file.

**Reply by herbert1984** [reply]
Thank you very much! Calculator restored, yay! Forever grateful for the quick help. Sorry for me being a dummie! Best regards

**Comment by farag2** [comment]
https://github.com/farag2/Windows-10-Sophia-Script#how-to-run-the-specific-functions
